index.py

```diff
@@ -198,6 +213,7 @@ def main():
 
             r = fetch("https://liveuamap.com/")
             html = BeautifulSoup(r, "html.parser")
+            _check_cloudflare(html)

```

Comment:
I recommend having this spit a variable instead, and then something like "If True is returned, throw error and wait time before doing a `continue` back to start"